examples/cloud-router-2-virtual-device-redundant-connection/main.tf

```diff
@@ -0,0 +1,31 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "cloud_router_virtual_device_redundant_connection" {
+  source = "../../modules/cloud-router-connection"
+
+  connection_name       = var.connection_name
+  connection_type       = var.connection_type
+  notifications_type    = var.notifications_type
+  notifications_emails  = var.notifications_emails
+  bandwidth             = var.bandwidth
+  purchase_order_number = var.purchase_order_number
+
+  #Aside
+  aside_ap_type  = var.aside_ap_type
+  aside_fcr_uuid = var.aside_fcr_uuid
+
+  #Zside
+  zside_ap_type        = var.zside_ap_type
+  zside_vd_type        = var.zside_vd_type
+  zside_vd_uuid        = var.zside_vd_uuid
+  zside_interface_id   = var.zside_interface_id
+  zside_interface_type = var.zside_interface_type
+
+  #Secondary-Connection
+  secondary_connection_name = var.secondary_connection_name
+  secondary_bandwidth       = var.secondary_bandwidth
+  zside_vd_sec_uuid         = var.zside_vd_sec_uuid
```

Comment:
There should be a secondary FCR uuid and not a second VD uuid. Always think of the modules as 2-1; NOT 1-2 or 2-2.

Reply:
The FCR-2NE-Redundant example supports using two virtual devices (Primary and secondary with different UUIDs), so we are following 1-2 approach. 
- confirmed with Alita Team, Gautham & Tim.